src/blocks/scribe-plugins/scribe-text-block-plugin.js

```diff
@@ -65,6 +67,33 @@ var ScribeTextBlockPlugin = function(block) {
       return (getTotalLength() === currentRange.end) && (currentRange.start === currentRange.end);
     };
 
+    var createBlocksFromParagraphs = function() {
+      var fakeContent = document.createElement('div');
+      fakeContent.appendChild(selectToEnd().extractContents());
+
+      stripFirstEmptyElement(fakeContent);
+
+      if (!scribe.allowsBlockElements()) {
+        var tempContent = document.createElement('div');
+        tempContent.appendChild(fakeContent);
+        fakeContent = tempContent;
+      }
```

Comment:
Why does wrapping in a div solve a problem if scribe.allowsBlockElements is false?